pallets/investments/src/lib.rs

```diff
@@ -1071,6 +1071,7 @@ where
 	}
 }
 
+// This impl seems to only be used for testing purposes.
 impl<T: Config> Investment<T::AccountId> for Pallet<T>
```

Comment:
Maybe the `Investment` trait is not needed because the test can call directly to the pallet (?).